core/app/[locale]/(default)/product/[slug]/_components/reviews.tsx

```diff
@@ -107,8 +132,8 @@ const getPaginationInfo = async (productId: number) => {
 
   return hasNextPage || hasPreviousPage
```

Comment:
Oversight on my part, but we should be using `pageInfoTransformer` for this, like the other components.

Reply:
Done. This also uncovered a bug where we weren't properly passing pagination button i18n labels